src/Sylius/Bundle/ApiBundle/Resources/config/serialization/OrderItem.xml

```diff
@@ -52,6 +52,12 @@
             <group>shop:order_item:read</group>
             <group>shop:cart:read</group>
         </attribute>
+        <attribute name="originalUnitPrice" serialized-name="originalPrice">
+            <group>admin:order:read</group>
+            <group>admin:order_item:read</group>
+            <group>shop:order_item:read</group>
+            <group>shop:cart:read</group>
+        </attribute>
```

Comment:
Is there any contract test checking this change in the API response?

Reply:
The serialization scope hasn't changed, we added to serialization `originalUnitPrice` field that earlier was included by the `OrderItemNormalizer` therefore I think there is no need to add the test